packages/cardpay-reward-programs/tests/rules/test_dummy_rule.py

```diff
@@ -0,0 +1,222 @@
+import pandas as pd
+import duckdb
+
+from cardpay_reward_programs.rules import dummy_rule
+
+def create_rule(
+    monkeypatch, fake_data_spend_acc, fake_data_safe_own, core_config_overrides={}, user_config_overrides={}
+):
+    
+    core_config = {
+        "payment_cycle_length": 32768,
+        "start_block": 0,
+        "end_block": 10000,
+        "subgraph_config_locations": {
+            "prepaid_card_payment": "s3://cardpay-staging-partitioned-graph-data/data/prepaid_card_payments/0.0.3/",
+            "spend_accumulation": "s3://tall-data-dev/paulin/spend_accumulation/0.0.1/",
+            "safe_owner": "s3://tall-data-dev/paulin/safe_owner/0.0.1/"  
+        }
+    }
+    core_config.update(core_config_overrides)
+    user_config = {
+        "token": "0x52031d287Bb58E26A379A7Fec2c84acB54f54fe3",
+        "base_reward": 5000000000000000000,
+        "transaction_factor": 2,
+        "spend_factor": 2,
+        "duration": 43200
+    }
+
+    user_config.update(user_config_overrides)
+    con = duckdb.connect(database=":memory:", read_only=False)
+
+    con.execute("create table SPEND_TABLE as select * from fake_data_spend_acc")
+    con.execute("create table SAFE_TABLE as select * from fake_data_safe_own")
+    print(con.fetchdf()) # It is not creating the table_
+
+    def table_query(
+        self, config_name, table_name, min_partition: int, max_partition: int
+    ):
+
+        if table_name == "safe_owner":
+            print("return safe")
```

Comment:
Handy for debug, but print statements should be mostly removed in the PR.

Reply:
Solved through commit cc318e856